modules/messages/src/lib.rs

```diff
@@ -255,6 +258,9 @@ decl_module! {
 		/// Deposit one of this module's events by using the default implementation.
 		fn deposit_event() = default;
 
+		/// Gets the chain id value from the instance.
+		const bridged_id: ChainId = T::BridgedChainId::get();
```

Comment:
I could see this name (`bridged_id`) in the metadata. Is that a common practice to use snake_case for identifying things in metadata? IIUC it is better to change it to `BridgedChainId` (perfectly fine to use `BridgedChainId` both for `Get`ter and this const).

Reply:
as the polkadotjs api transforms automatically from snake_case to camel_case i thought using snake_case was correct for this. I'm afraid that i don't have enough expertise to determine if this is a common practice or not. I prefer following your recommendation and switch to `BridgedChainId `